client/components/head/index.jsx

```diff
@@ -3,14 +3,49 @@
  *
  * @format
  */
-
 import React from 'react';
 import PropTypes from 'prop-types';
+import { get, isEmpty } from 'lodash';
 
 /**
  * Internal dependencies
  */
 
+/**
+ * This file is sourced early and cannot easily be connected to the state tree...
+ * so, this function consults the value in localStorage.
+ *
+ * @returns {string} The preference setting that exists in localStorage (if any)
+ */
+function getColorSchemePreferenceFromLocalStorage() {
+	try {
+		const prefs = localStorage.getItem( 'preferences-data' );
+		if ( isEmpty( prefs ) ) {
+			return 'default';
+		}
+		const parsed = JSON.parse( prefs );
+		return get( parsed, 'colorScheme', 'default' );
+	} catch ( err ) {
+		return 'default';
+	}
+}
+
+/**
+ * This function is used in setting the "theme_color" head | meta value.
+ * It is used to customize the (browser UI) color for mobile Chrome, Opera, etc. on Android
+ *
+ * @see https://developers.google.com/web/fundamentals/design-and-ux/browser-customization/#meta_theme_color_for_chrome_and_opera
+ * @returns {string} The color for the selected color scheme
+ */
+function getBrowserThemeColor() {
+	const DEFAULT_COLOR = '#016087'; // blue-500
+	const colors = {
+		'classic-blue': DEFAULT_COLOR,
+		'classic-bright': DEFAULT_COLOR,
+	};
+	return get( colors, getColorSchemePreferenceFromLocalStorage(), DEFAULT_COLOR );
```

Comment:
it won't matter yet since all the colors are the same, but this won't react to changes when the user picks a new theme will it? should it?

Reply:
Correct, this won't react to changes -- I'm not sure if there's a way to make it re-evaluate the header without a full pageload. 

It's less than ideal, but I think it's ok-ish.